ControlFlow/Program.cs

```diff
@@ -15,22 +15,33 @@ public static class ControlFlow
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
         {
-            return default;
+            if (number > 1 && number < 10)
+                return "Valid";
+            else
+                return "Invalid";
         }
 
         // 1.2 - Write a method which takes two integers and returns the maximum of the two. If they
         // are the same, return either one.
-        public static int Maximum(int number1, int number2)
+        public static object Maximum(int number1, int number2)
```

Comment:
Just bringing this to your attention because I am not sure if you realized you did this and I am not going to make you have to change this, however in the future changing the method signature of a program is NOT allowed unless you have a specific reason to. That reason would need to brought up with us because there could be a compiler error. For example if our tests were looking for strings and you changed the output type to an int.